docs/src/install/database.rst

```diff
@@ -79,7 +76,7 @@ stage.
    $ orion db test
 
    Check for a configuration inside the default paths...
-       {'type': 'mongodb', 'name': 'mydb', 'host': 'localhost'}
+       {'type': 'pickleddb', 'name': 'orion', 'host': 'orion_db.pkl'}
```

Comment:
You can drop `'name'` for the pickleddb.

Reply:
Actually I would add 'port' since it would match what orion outputs.

I agree that they are irrelevant for pickleddb, but it might confuse new users to see unexpected output.